### PR TITLE
#164069315 like and dislikes articles 

### PR DESCRIPTION
**Pull request by akram256**
**What does this PR do?**
* Implements like and dislike of an article
* Write tests for like and dislike of articles

**Description of the tasks to be completed?**
* Create a liking_disliking file in article app
* Updates models file in article app
* Updates ArticleSerializer in serializers file of the article app
* Updates views file in article app

**How should this be manaully tested?**
* Git checkout `ft-like-and-dislike-artcles-164069315` in your terminal to switch onto your branch
* Run `python manage.py makemigrations`
* Run `python manage.py migrate`
* Then you can run your server with `python manage.py migrate`
**Don't forget to register, verify account then log in**
**Then copy and paste the token to headers, Authorization, choose Bearer Token**
**Create article**
* Post `/api/articles/` to create articles
Screenshot 2019-03-11 at 15 04 53

* To like an article `/api/articles/slug/like`

Screenshot 2019-03-11 at 15 05 49

*To dislike an article `/api/articles/slug/dislike/`

Screenshot 2019-03-11 at 15 08 07

***What are the relevant pivot tracker stores?**
#164069315

### CODE REVIEW

**Review by nyabongoedgar — APPROVED** [review]
Good job

**Review by Stanley-Okwii — APPROVED** [review]

File: authors/apps/articles/urls.py [hunk 1]
```diff
@@ -3,13 +3,26 @@
...
+
 
 urlpatterns = [
     path('categories/', CreateListCategory.as_view(), name='create-category'),
     path('categories/<str:slug>', RetrieveUpdateDestroyCategory.as_view(),
          name='update-delete'),
     path('articles/', CreateArticle.as_view(), name='create-articles'),
     path('article/<str:slug>/', ArticleRetrieveUpdate.as_view(),
-         name='update-articles')
+         name='update-articles'),
```
Comment (Stanley-Okwii) [inline authors/apps/articles/urls.py:17]: @akram256 , Use the article slug not 'ID' in your endpoints. Most of the API spec on the readme use the same thing.

**Review by Stanley-Okwii — APPROVED** [review]

File: authors/apps/articles/urls.py (hunk 1 above)
Comment (akram256) [reply authors/apps/articles/urls.py:17]: on it

Comment (akram256) [reply authors/apps/articles/urls.py:17]: Hey @Stanley-Okwii , I have worked on the feedback. Thank you

**Review by NLSanyu — APPROVED** [review]

**Review by walimike — APPROVED** [review]

**Review by PatrickCmd — APPROVED** [review]

**Pull request merged**